e2e/README.md

```diff
@@ -113,20 +130,6 @@ assertions on the contents of both the stdout and stderr streams that result fro
 validators can be used to assert on numerous types of properties of the live VM, such as the live file system and kernel
 state.
 
-You can find all implemented scenarios in the [scenario](scenario/) pacakge within files prefixed with `scenario_`.
-The `Scenario` struct definition can be found in [scenario/types.go](scenario/types.go).
-
-### Implementation
```

Comment:
I'd opt for keeping this section around and updating it with correct instructions

Reply:
I thought about it. And my current suggestion would be "copy any test and make some tweaks".

Which is probably isn't really helpful and can be wrong at some stage in the future.